### ISSUE TITLE
The ARIA form field's name property does not contain all of its instructions in Wizards tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1917]
URL: http://clarity-ks.surge.sh/wizards

Tools: Web Accessibility Toolbar (WAT2.0) / ARIA Markup

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Wizards tab.
3. Click OPEN WIZARD button for Form validation.
4. Use WAT to check the ARIA form’s name property.

Actual Results:
The ARIA form field's name property only contain its own visual instructions.
For example, a form section for "Form with validation" and "Favorite food " is an input text within that section. The name property for “Favorite food” only include "Favorite food."
For more details, please refer to "Wizards-ARIAFromNameUncomplete.png".

Expected Results:
The ARIA form field's name property should contain all of its instructions.
For example, the name property for “Favorite food” should include both " Form with validation" and "Favorite food."

Note:
This issue also occurs on Wizard with form validation window, Ghost Pages window.

Violation of Section 508 Standards: 
1194.22 (n) When electronic forms are designed to be completed on-line, the form shall allow people using assistive technology to access the information, field elements, and functionality required for completion and submission of the form, including all directions and cues.

User Impact: Blind.

Screen shot:
1.	Wizards-ARIAFromNameUncomplete.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/c1c8894c-37b8-43a6-b7cd-4610e4ae4e15)

NOTE: this may also be an issue on the website
